code/modules/mob/living/basic/icemoon/wolf/wolf_ai.dm

```diff
@@ -28,6 +28,7 @@
 		/datum/ai_planning_subtree/simple_find_target,
 		/datum/ai_planning_subtree/attack_obstacle_in_path,
 		/datum/ai_planning_subtree/basic_melee_attack_subtree,
+		/datum/ai_planning_subtree/find_and_hunt_target/look_for_functional_pipes,
```

Comment:
btw u should probably make sure tamed animals dont hunt down pipes.

Reply:
What Ben10 said, tamed animals shouldn't hunt the pipes, also it would be nice if these ai controller datums didn't have this planning subtree until the extraction machine is spawned, since there's no guarantee it'll be built in a given round, especially on icebox where it isn't implemented yet. This could be achieved by registering a global signal that's send whenever it's initialized and by having a global var for mobs spawned afterward. It's perhaps a micro-op but what do you say, @Ben10Omintrix ?